Fitness/Views/GymFacilitiesView.swift

```diff
@@ -0,0 +1,483 @@
+            ("10 AM - 12 AM", "WED"),
+            ("10 AM - 12 AM", "THU"),
+            ("10 AM - 12 AM, 2 PM - 12 AM", "FRI"),
+            ("10 AM - 12 AM, 2 PM - 12 AM", "SAT"),
+            ("10 AM - 12 AM", "SUN")
+            ]),
+        FacilityData(facility: "BOWLING LANES", isDropped: false, data: [
+            ("5:30 PM - 10 PM", "MON"),
+            ("5:30 PM - 11 PM", "TUE"),
+            ("8 PM - 11 PM", "WED"),
+            ("5:30 PM - 11 PM", "THU"),
+            ("5 PM - 11:30 PM", "FRI"),
+            ("12 AM - 11:30 PM", "SAT"),
+            ("CLOSED", "SUN")
+            ]),
+        FacilityData(facility: "SWIMMING POOL", isDropped: false, data: [
+            ("7 AM - 8:45 AM, 11 AM - 1:30 PM, 5 PM - 6:30 PM", "MON"),
+            ("6 AM - 8:45 AM, 11 AM - 1:30 PM, 5 - 6:30 PM, 10 - 11:15 PM", "TUE"),
+            ("7 AM - 7:45 AM, 8 AM - 8:45 AM, 11 AM - 1:30 PM, 5 PM - 6:30 PM, 8:30 PM - 10 PM", "WED"),
+            ("6 AM- 8:45 AM, 11 AM - 1:30 PM, 5 - 6:30 PM, 10 - 11:15 PM", "THU"),
+            ("7 AM - 8:45 AM, 11 AM - 1:30 PM, 2 PM - 3: 15 PM, 5:30 PM - 7 PM", "FRI"),
+            ("2 PM - 4 PM", "SAT"),
+            ("CLOSED", "SUN")
+            ])
+    ]
+    let teagleFacilities: [FacilityData] = [
+        FacilityData(facility: "SWIMMING POOL", isDropped: false, data: [
+            ("9:45 AM - 11:15 AM, 12:15 PM - 1:30 PM", "MON"),
+            ("6 AM - 8:30 AM, 9:45 AM - 11:15 AM, 12:15 PM - 1:30 PM", "TUE"),
+            ("6 AM - 8:30 AM, 8:45 AM - 11:15 AM, 12:15 PM - 1:30 PM", "WED"),
+            ("9:45 AM - 11:15 AM, 12:15 PM - 1:30 PM", "THU"),
+            ("6 AM - 8:30 AM, 9:45 AM - 11:15 AM, 12:15 PM - 1:30 PM", "FRI"),
+            ("CLOSED", "SAT"),
+            ("12 AM - 2 PM", "SUN")
+            ])
+    ]
+    var facilityName: FacilityName!
+    var facilityData: [FacilityData]!
+    var facilityTableViews: [UITableView] = []
+    
+    var bowlingTableView: UITableView?
+    var poolTableView: UITableView?
+    var basketballTableView: UITableView?
+    var basketballSeparator: UIView?
+    var poolSeparator: UIView?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+    }
+    
+    convenience init(facilityName: FacilityName) {
+        self.init(frame: .zero)
+        self.facilityName = facilityName
+        switch facilityName {
+        case .helenNewman:
+            facilityData = helenNewmanFacilities
+        case .teagle:
+            facilityData = teagleFacilities
+        case .noyes, .appel:
+            facilityData = []
+        }
+        
+        setupViews(facilityName: facilityName)
+        setupConstraints()
+    }
+    
+    private func setupViews(facilityName: FacilityName) {
+        guard let unwrappedFacilityName = self.facilityName else {
+            return
+        }
+        
+        switch unwrappedFacilityName {
+        case .helenNewman:
+            bowlingTableView = UITableView(frame: .zero, style: .grouped)
```

Comment:
How come we need so many table views in one view? Would sections not be enough with different cell classes?

Reply:
This is so obvious why did I miss this? 😑 I even consulted. Ok. Yeah that makes so much more sense than this multiple tableview nonsense - sections literally is so much simpler. Wow how did I not even think of that after thinking of so many other ways? Going to fix, and will probably result in significant changes and fixes as well.

Reply:
No problem! That's what these reviews are for!